libs/single-spa-angular/package.json

```diff
@@ -24,8 +24,9 @@
   "homepage": "https://github.com/single-spa/single-spa-angular#readme",
   "sideEffects": false,
   "peerDependencies": {
-    "@angular/core": ">=12",
+    "@angular/core": ">=13",
     "json5": "*",
-    "single-spa": ">=4.0.0"
+    "single-spa": ">=4.0.0",
+    "style-loader": "^3.3.1"
```

Comment:
This doesn't look like a good thing to do - new versions of npm autoinstall peer dependencies rather than using them, so this might not actually end up with the intended behavior. Instead of using Angular CLI's version of style-loader, it could be a different one.

Reply:
@arturovt I think this is a valid comment, what is the reason behind this new dependency?

Reply:
https://github.com/angular/angular-cli/pull/21470/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L214

Reply:
Ah ok, Angular Cli removed it then, because we need it for injecting CSS when loading the micro-frontend, you add it again.

Thx for the precisions @arturovt